main.go

```diff
@@ -160,12 +160,14 @@ func main() {
 		}
 	}
 	// Fetch the configuration for maximum number of API keys allowed per user.
-	maxAPIKeys, err := strconv.Atoi(os.Getenv(envMaxNumAPIKeysPerUser))
-	if err != nil {
-		log.Printf("Warning: Failed to parse %s env var. Error: %s", envMaxNumAPIKeysPerUser, err.Error())
-	}
-	if maxAPIKeys > 0 {
-		database.MaxNumAPIKeysPerUser = maxAPIKeys
+	if maxAPIKeysStr := os.Getenv(envMaxNumAPIKeysPerUser); maxAPIKeysStr != "" {
```

Comment:
I'd use `LookupEnv` here. That way you can check if the user had the intention to set the environment variable and you can return an error if the user set it to an empty string by accident.

Reply:
Hmm. Am I alone in thinking that when I set and env var to "" it's the same as if it's gone? I mean, I know that it's detectable but I usually presume that the software will ignore the empty value.

That said, I don't mind. This will print a warning but it won't block. I'll change it.